CarRental.Domain/Data/DataSeed.cs

```diff
@@ -0,0 +1,108 @@
+﻿using CarRental.Domain.Models;
+
+namespace CarRental.Domain.Data;
+
+/// <summary>
+/// Represents a set of test data for car rentals.
+/// </summary>
+public class DataSeed
+{
+    /// <summary>List of car models.</summary>
+    public List<Model> Models { get; }
+
+    /// <summary>List of car model generations.</summary>
+    public List<ModelGeneration> Generations { get; }
+
+    /// <summary>List of cars available in the rental system.</summary>
+    public List<Car> Cars { get; }
+
+    /// <summary>List of clients who can rent cars.</summary>
+    public List<Client> Clients { get; }
+
+    /// <summary>List of car rentals performed by clients.</summary>
+    public List<Rental> Rentals { get; }
+
+    /// <summary>
+    /// Initializes the test data collections with diverse data for cars, clients, generations, and rentals.
+    /// </summary>
+    public DataSeed()
+    {
+        Models = new List<Model>
+        {
+            new() { Name = "Toyota Corolla", DriveType = "FWD", Seats = 5, BodyType = "Sedan", Class = "C" },
+            new() { Name = "Ford Focus", DriveType = "FWD", Seats = 5, BodyType = "Hatchback", Class = "C" },
+            new() { Name = "BMW X5", DriveType = "AWD", Seats = 5, BodyType = "SUV", Class = "E" },
+            new() { Name = "Kia Rio", DriveType = "FWD", Seats = 5, BodyType = "Sedan", Class = "B" },
+            new() { Name = "Mercedes E-Class", DriveType = "RWD", Seats = 5, BodyType = "Sedan", Class = "E" },
+            new() { Name = "Audi A4", DriveType = "AWD", Seats = 5, BodyType = "Sedan", Class = "D" },
+            new() { Name = "Hyundai Tucson", DriveType = "AWD", Seats = 5, BodyType = "SUV", Class = "C" },
+            new() { Name = "Volkswagen Golf", DriveType = "FWD", Seats = 5, BodyType = "Hatchback", Class = "C" },
+            new() { Name = "Nissan Qashqai", DriveType = "AWD", Seats = 5, BodyType = "SUV", Class = "C" },
+            new() { Name = "Honda Civic", DriveType = "FWD", Seats = 5, BodyType = "Sedan", Class = "B" }
+        };
```

Comment:
Упростить инициализацию коллекций. Тут и далее по коду.

```suggestion
        Models =
        [
            new() { Name = "Toyota Corolla", DriveType = "FWD", Seats = 5, BodyType = "Sedan", Class = "C" },
            new() { Name = "Ford Focus", DriveType = "FWD", Seats = 5, BodyType = "Hatchback", Class = "C" },
            new() { Name = "BMW X5", DriveType = "AWD", Seats = 5, BodyType = "SUV", Class = "E" },
            new() { Name = "Kia Rio", DriveType = "FWD", Seats = 5, BodyType = "Sedan", Class = "B" },
            new() { Name = "Mercedes E-Class", DriveType = "RWD", Seats = 5, BodyType = "Sedan", Class = "E" },
            new() { Name = "Audi A4", DriveType = "AWD", Seats = 5, BodyType = "Sedan", Class = "D" },
            new() { Name = "Hyundai Tucson", DriveType = "AWD", Seats = 5, BodyType = "SUV", Class = "C" },
            new() { Name = "Volkswagen Golf", DriveType = "FWD", Seats = 5, BodyType = "Hatchback", Class = "C" },
            new() { Name = "Nissan Qashqai", DriveType = "AWD", Seats = 5, BodyType = "SUV", Class = "C" },
            new() { Name = "Honda Civic", DriveType = "FWD", Seats = 5, BodyType = "Sedan", Class = "B" }
        ];
```